<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chart</class>
 <widget class="QDialog" name="chart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>虎图帐</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img.qrc">
    <normaloff>:/img/icon.jpg</normaloff>:/img/icon.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#chart{
	background-color: #fafafa;
}
QLabel {
	font-family: &quot;微软雅黑&quot;;
	font-size: 18px;
	color: #666;
}

QPushButton#AccountStatementBtn,QPushButton#AccountingBtn,QPushButton#ChartBtn,QPushButton#HomepageBtn {
    color: #fff;
    font-size: 45px;
    font-family: &quot;逐浪雅宋体&quot;;
    line-height: 80px;
    height: 80px;
    border: 0;
}

QPushButton#AccountStatementBtn:hover,QPushButton#AccountingBtn:hover,QPushButton#ChartBtn:hover,QPushButton#HomepageBtn:hover {
	
	background-image: url(:/img/hoverBackground.png);
}
QPushButton#AccountStatementBtn:pressed,QPushButton#AccountingBtn:pressed,QPushButton#ChartBtn:pressed,QPushButton#HomepageBtn:pressed {
	
	background-image: url(:/img/pressedBackground.png);
}


QWidget#leftContainer {
    background-color:#ffbc2e
}
QComboBox {
	line-height: 20px;
    height: 20px;
    font-family: &quot;微软雅黑&quot;;
    font-size: 18px;
    color: #666;
	border: 1px solid #ccc;
}

QComboBox::down-arrow {
	width: 16px;
   
	image: url(:/img/btnDownUnChecked.png);
	background-color: #fff;
}
QComboBox::down-arrow:pressed{
   
	image: url(:/img/btnDownChecked.png);
}
QComboBox::drop-down {
	border: 0;
}
#setBtn {
	border: 0;
	background-image: url(:/img/keyUnclicked.png);
}
#setBtn:pressed {
	background-image :url(:/img/keyCLicked.png)
}</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>161</width>
     <height>561</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="lOuterLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="leftContainer" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>131</width>
         <height>501</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="leftLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="HomepageBtn">
          <property name="text">
           <string>首页</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="AccountingBtn">
          <property name="text">
           <string>记账</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="AccountStatementBtn">
          <property name="text">
           <string>流水</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ChartBtn">
          <property name="text">
           <string>图析</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>675</width>
     <height>481</height>
    </rect>
   </property>
   <widget class="QComboBox" name="chooseYear">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <item>
     <property name="text">
      <string>2015</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2016</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2017</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2018</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2019</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2020</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="chooseChart">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>221</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>收入支出对比（柱状图）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>分类收入对比（柱状图）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>分类支出对比（柱状图）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>收入支出分类饼状图</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="chooseMonth">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>全年</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12</string>
     </property>
    </item>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>631</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="chartLayout">
     <property name="spacing">
      <number>0</number>
     </property>
    </layout>
   </widget>
   <widget class="QLabel" name="timeLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>55</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>时间：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>报表类型：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>月</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>年</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>490</y>
     <width>59</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="img.qrc">:/img/earnList.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="setBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>450</y>
     <width>159</width>
     <height>53</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>97</width>
     <height>56</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="outcomeLabel">
   <property name="geometry">
    <rect>
     <x>345</x>
     <y>120</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string> 本月支出分类饼状图</string>
   </property>
  </widget>
  <widget class="QLabel" name="incomeLabel">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>120</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>本月收入分类饼状图</string>
   </property>
  </widget>
  <widget class="QLabel" name="outcomeInfo">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>400</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="img.qrc">:/img/earnList.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="incomeInfo">
   <property name="geometry">
    <rect>
     <x>635</x>
     <y>400</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="img.qrc">:/img/payList.png</pixmap>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
